### ISSUE TITLE
Deprecate Schema & Properties Id field

### ISSUE THREAD

**Issue by GCHQDeveloper404** [gchq/Gaffer#1477]
Related to issues and questions raised from #1472.

Adding Schema/Props to GraphLibrary required the modification of the Id field. This would/could lead to equality issues after adding a Schema/Props as the Json stored would be different to what was given.

This ticket is to remove/depreciate Id fields and add the Id when adding to the GraphLibrary. This removes some of the need to maintain ID's from #1431.

The fields are still required when adding to GraphLibrary so that the Schema/Props keyed with same Id save on storage space.

**Comment by GCHQDeveloper404** [comment]
waiting on #1472

**Comment by p013570** [comment]
This will be done as part of #1472.

**Comment by p013570** [comment]
Merged into develop.
